src/lib.rs

```diff
@@ -113,7 +117,18 @@ impl VadSession {
 
     /// Pass in some audio to the VAD and return a list of any speech transitions that happened
     /// during the segment.
+    ///
+    /// If "auto_resample" feature is enabled, then this function will resample what ever it
```

Comment:
Update feature name in doc comment. "what ever" should be "whatever". And line below change "then user input is expected to be of the right" to "user input is expected to be either" and maybe mention the unit is hertz/Hz